torchvision/ops/boxes.py

```diff
@@ -5,7 +5,7 @@
 
 
 @torch.jit.script
-def nms(boxes, scores, iou_threshold):
+def nms(boxes: Tensor, scores: Tensor, iou_threshold: float) -> Tensor:
     # type: (Tensor, Tensor, float) -> Tensor
```

Comment:
Can you remove the comment-style type annotations if you add the type annotations inline?

Reply:
Sure, I have it in the local repo, will push it soon.